ui/src/parseError.ts

```diff
@@ -113,6 +117,15 @@ function parseIfHtml(message: string): string {
     return message;
 }
 
+function parseIfXml(message: string): string {
+    const matches: RegExpMatchArray | null = message.match(/<\?xml.*<Message>(.*\nRequestId:.*\nTime:.*)<\/Message>/);
```

Comment:
The RequestId stuff is a little too specific for me. Part of why I wanted the whole "Message" property is so that this code would work even if they changed the format of the message in the future. Perhaps you did this because you had problems matching a multi-line string? If so, try the "s" flag, which means `.` will also match new lines.

Here's the simplest regex that still gets the job done, IMO:
```
/<Message>(.*)<\/Message>/si
```

(I threw in the "i" flag to handle if "message" is lowercase)

Reply:
I didn't realize that 's' flag was an option, looks very clean 🧼